src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -100,4 +112,11 @@ public String listPlaystation(Model theModel) {
 		
 		return "list-playstation";
 	}
+	
+	@GetMapping("/xbox")
+	public String listXbox(Model theModel) {
+		theModel.addAttribute("xbox", listXbox);
+		
+		return "list-xbox";
+	}
```

Comment:
I mapped the new Xbox list to the endpoint /xbox and returned the new list-xbox html page I created, similar to how your other pages are mapped